Src/Support/Google.Apis.Auth/OAuth2/AwsExternalAccountCredential.AwsSecurityCredential.cs

```diff
@@ -0,0 +1,144 @@
+﻿/*
+Copyright 2022 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+using Google.Apis.Json;
+using Google.Apis.Util;
+using System;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace Google.Apis.Auth.OAuth2
+{
+    public partial class AwsExternalAccountCredential
+    {
+        /// <summary>
+        /// Represents AWS security credentials which are used to sign
+        /// the subject token.
+        /// </summary>
+        private sealed class AwsSecurityCredentials
+        {
+            private const string AccessKeyIdEnvVar = "ACCESS_ACCESS_KEY_ID";
```

Comment:
I believe this should be AWS_ACCESS_KEY_ID, and that the AIP is wrong.
https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html shows it as AWS_ACCESS_KEY_ID.